java/google/registry/rde/imports/RdeContactImportAction.java

```diff
@@ -120,8 +126,33 @@ private RdeImportUtils createRdeImportUtils() {
     }
 
     @Override
-    public void map(ContactResource contact) {
-      getImportUtils().importContact(contact);
+    public void map(JaxbFragment<XjcRdeContactElement> fragment) {
+      final XjcRdeContact xjccontact = fragment.getInstance().getValue();
+      try {
+        logger.infofmt("Converting xml for contact %s", xjccontact.getId());
+        // Record number of attempted map operations
+        getContext().incrementCounter("contact-import-attempted");
```

Comment:
Mapreduce counters are free-form strings that are formatted like "%d %s".  We have a convention in our codebase of using normal English phrases for readability, e.g. "contact imports attempted".  Same for the others.